### PR TITLE
increase max battery charge gauge threshold to 101 in case of LFP

### LINKED ISSUE #4184: battery level gauge shows red value if battery is 100% and LFP is selected
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

In case of LFP Tesla, that is, lfp option is flagged in teslamate config, the battery level gauge correctly shows different alert thresholds, however, the edge case where the battery is actually 100% charged still shows as red.
It's probably enough to change the limit config query to use "101" as max level in case of LFP. See screenshot for proposed solution.

### Expected Behavior

See screenshot

### Steps To Reproduce

_No response_

### Relevant log output

```shell
See screenshot
```

### Screenshots

issue with 100 percent

proposed change

### Additional data

_No response_

### Type of installation

Docker

### Version

v1.30.1

### PR DESCRIPTION
**Pull request by neothematrix**
fixes #4184: visualization issue of gauge turning "red" in case of 100% battery charged on lfp teslas

### CODE REVIEW

File: grafana/dashboards/overview.json [hunk 1]
```diff
@@ -2169,4 +2169,4 @@
   "uid": "kOuP_Fggz",
   "version": 9,
```
Comment (JakobLichterfeld) [inline grafana/dashboards/overview.json:2170]: version should be updated

**Review by JakobLichterfeld — CHANGES_REQUESTED** [review]
Thanks for your suggestion.

File: grafana/dashboards/overview.json (hunk 1 above)
Comment (neothematrix) [reply grafana/dashboards/overview.json]: @JakobLichterfeld  thanks, I missed it, new commit is ready

**Review by JakobLichterfeld — APPROVED** [review]

**Comment by swiffer** [comment]
looks good!

**Pull request merged**